fairlearn/__init__.py

```diff
@@ -23,8 +23,7 @@
     handler = logging.FileHandler(fairlearn_logs, mode="w")
     handler.setLevel(logging.INFO)
     logger.addHandler(handler)
-    logger.info("Initializing logging file for fairlearn")
-
+    
```

Comment:
I think that the issue meant to get rid of all the logic around `fairlearn_logs`